backend/src/main/resources/db/migration/V002__add_status_column.sql

```diff
@@ -0,0 +1,6 @@
+create type status as enum ('DRAFT', 'PUBLISHED');
+
+alter table domain
+add column status status not null default 'DRAFT'
```

Comment:
Would you want the existing ones set to Draft? Or are they effectively Published?

Reply:
There's no real data at the moment so yeah, we'll default to draft when a new record is created. I'll be updating the setup (that generates test data) later.